core/src/muxing.rs

```diff
@@ -160,11 +160,19 @@ pub trait StreamMuxerExt: StreamMuxer + Sized {
     }
 
     /// Returns a future that resolves to the next inbound `Substream` opened by the remote.
+    #[deprecated(
```

Comment:
Good catch. Would you mind extracting this change into a separate pull request?

Reply:
Done: https://github.com/libp2p/rust-libp2p/pull/3002